src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -42,6 +44,7 @@
 			>
 				<IconTwitter />
 			</ExternalLinkIcon>
+			<Badge variant="warning">{APP_VERSION}</Badge>
```

Comment:
was it decided without `v.` in front? usually it is a good indicator of what the numbers refer too

Reply:
I made this implementation based on this conversation here: https://dfinity.slack.com/archives/C072H2FMW1F/p1729777449248549?thread_ts=1729774145.798829&cid=C072H2FMW1F

I'm not sure if there is a decision yet.